diff -ur autoconf-2.71/bin/autom4te.in autoconf-2.71_patched/bin/autom4te.in
--- autoconf-2.71/bin/autom4te.in	2021-01-28 15:46:48
+++ autoconf-2.71_patched/bin/autom4te.in	2022-12-08 16:21:03
@@ -260,6 +260,7 @@
 {
   my ($file) = @_;
   use Text::ParseWords;
+  use File::Basename;
 
   my $cfg = new Autom4te::XFile ($file, "<");
   my $lang;
@@ -271,7 +272,10 @@
 	if /^\s*(\#.*)?$/;
 
       my @words = shellwords ($_);
+
       my $type = shift @words;
+      my $dirname = dirname(__FILE__);
+      @words = map { s!^@pkgdatadir@!$dirname/../share/autoconf!; $_ } @words;
       if ($type eq 'begin-language:')
 	{
 	  fatal "$file:$.: end-language missing for: $lang"
diff -ur autoconf-2.71/lib/autom4te.in autoconf-2.71_patched/lib/autom4te.in
--- autoconf-2.71/lib/autom4te.in	2021-01-28 15:46:48
+++ autoconf-2.71_patched/lib/autom4te.in	2022-12-09 13:59:43
@@ -116,7 +116,7 @@
 
 begin-language: "Autoconf-without-aclocal-m4"
 args: --prepend-include '@pkgdatadir@'
-args: --cache=autom4te.cache
+args: --no-cache
 args: autoconf/autoconf.m4f
 args: acsite.m4?
 args: --mode 777
@@ -142,7 +142,7 @@
 ## -------- ##
 
 begin-language: "Autotest"
-args: --prepend-include '@pkgdatadir@'
+args: --prepend-include ''@pkgdatadir@''
 args: autotest/autotest.m4f
 args: package.m4?
 args: local.at?
@@ -156,7 +156,7 @@
 ## ---- ##
 
 begin-language: "M4sh"
-args: --prepend-include '@pkgdatadir@'
+args: --prepend-include ''@pkgdatadir@''
 args: m4sugar/m4sh.m4f
 args: --mode 777
 args: --language M4sugar
@@ -168,6 +168,6 @@
 ## ------- ##
 
 begin-language: "M4sugar"
-args: --prepend-include '@pkgdatadir@'
+args: --prepend-include ''@pkgdatadir@''
 args: m4sugar/m4sugar.m4f
 end-language: "M4sugar"
