diff -ur autoconf-2.71/bin/autom4te.in autoconf-2.71_patched/bin/autom4te.in
--- autoconf-2.71/bin/autom4te.in	2021-01-28 15:46:48
+++ autoconf-2.71_patched/bin/autom4te.in	2022-12-08 16:21:03
@@ -260,6 +260,7 @@
 {
   my ($file) = @_;
   use Text::ParseWords;
+  use File::Basename;
 
   my $cfg = new Autom4te::XFile ($file, "<");
   my $lang;
@@ -271,7 +272,10 @@
 	if /^\s*(\#.*)?$/;
 
       my @words = shellwords ($_);
+
       my $type = shift @words;
+      my $dirname = dirname(__FILE__);
+      @words = map { s!^@pkgdatadir@!$dirname/../share/autoconf!; $_ } @words;
       if ($type eq 'begin-language:')
 	{
 	  fatal "$file:$.: end-language missing for: $lang"
